scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -37,6 +37,7 @@
 parser.add_argument("-p", "--postfix", type=str, help="Postfix for output file name", default=None)
 parser.add_argument("--eta", nargs=3, type=float, help="Eta binning as 'nbins min max' (only uniform for now)", default=[48,-2.4,2.4])
 parser.add_argument("--pt", nargs=3, type=float, help="Pt binning as 'nbins,min,max' (only uniform for now)", default=[29,26.,55.])
+parser.add_argument("--lumiUncertainty", type=float, help="Uncertainty for luminosity in excess to 1 (e.g. 1.012 means 1.2\%)", default=1.012)
```

Comment:
Why does this need a CL arg...?

Reply:
Well in principle one can use the same code for other years

Reply:
In this case the treatment would have to be extended to account for the correlation model in any case.

Reply:
Well ok, there is a default argument with the value we want, if we ever change the value one can just change it through the option. The final result is the same, but this seems less hardcoded with magic numbers inside the code